### DISCUSSION TITLE
GitHub hosted larger runners never start

### DISCUSSION THREAD

**Discussion by markgoho** [community/community#35490] [Actions] [answered]
I've created a new GitHub hosted runner (our org was just added to the beta test). When I update my workflow to use the runner label, the jobs all just hang on `Waiting for a runner to pick up this job...`

I think I've checked all the correct boxes and I still can't get this to work. What gives?

**Comment by markgoho** [comment]
Not a super great beta testing experience -- 6 days after having gotten access, this still doesn't work.

**Comment by rudolphfroger-ds** [comment]
Same with me, the larger runner is never started no matter how I configure labels, `runs-on` and runner groups.

**Reply by orjan** [reply]
Same here it's runs provisioning and then ends up in shutdown and is not picking up any work.

The first time I tried it out the runner actually started to do some work but the feedback to the job wasn't working.
After that I got a 500 error for the runner page and then it has not started again (even if I have created new runners with different labels)

**Comment by lkfortuna** [comment, marked as answer]
Hi there - have you tried [checking your spending limit](https://docs.github.com/en/billing/managing-billing-for-github-actions/managing-your-spending-limit-for-github-actions) set for Actions? If this is $0 the larger hosted runner jobs will not run.

**Reply by markgoho** [reply]
![image](https://user-images.githubusercontent.com/9759954/195919078-77011fad-7914-410d-8e29-0633dafa53c7.png)
![image](https://user-images.githubusercontent.com/9759954/195919133-88069cfa-dd1b-4f40-8030-d6ddaea3432c.png)

**Reply by markgoho** [reply]
@lkfortuna I _did_ have a $1.00 limit set. But then I changed it to $100 and I'm still getting the message above.

**Reply by lkfortuna** [reply]
@markgoho - Thanks for checking. Can you open a support ticket using the link in the message? The team can check it out from there to see what's going on more specifically. Thank you for bringing this to our attention!

**Reply by lkfortuna** [reply]
Feel free to post the link to the ticket here once you have

**Reply by markgoho** [reply]
https://support.github.com/ticket/personal/0/1833447

**Reply by markgoho** [reply]
It's working now! 
![image](https://user-images.githubusercontent.com/9759954/195945816-877f0219-8735-452c-b181-fe03a4397abf.png)

I think there might have just been a delay in increasing the billing limit and the runners being allowed to run.

**Reply by lkfortuna** [reply]
great news! Glad it's resolved.

**Comment by shinofara** [comment]
I have the same error.

**Comment by zackees** [comment]
Me too. I have the exact same error!

None of my windows runners are running across projects! Can an SRE please clear my job queue? Jobs are running fine for mac and ubuntu.

**Comment by bonesoul** [comment]
same error. how can i get it fixed?

**Comment by zackees** [comment]
I found out the problem. I was using an invalid builder name for one of the windows projects. I think I used `win-latest` instead of `windows-latest` and so github failed to find a builder with that name and just never ran.

Github should do something like if not valid name then error, but they don't.

**Comment by kodyline** [comment]
Tried so many things, groups, labels, setting unlimited spending limits but still always met with:
`Waiting for a runner to pick up this job...`

**Reply by kodyline** [reply]
For anyone reading this:
You have two levels of needing to put the repositories on the allow list at the organization level (not JUST the enterprise level)

So create them at the enterprise level **THEN** select what repos can use it at the **ORGANIZATION** level.

**Comment by paintoshi** [comment]
I have a different kind of problem. The runner itself says "There are currently no running jobs" when I have a running job running. I know it's using the label "myrunner" correct because the job is called "Runner name: myrunner_12b250132f2e"

**Comment by birkskyum** [comment]
I have two large github hosted runners, an 8-core and a 16-core ubuntu, in a runner group. (github org `maplibre`)

When the 8 core ubuntu is assigned, it'll run the actions.
When the 16 core ubuntu is assigned, it'll be stuck at `Job is waiting for a runner from <my large gh runner name> to come online.`

I have ample room in the spending limit.

**Comment by trubi** [comment]
Any idea what is going on here? I have the same issue but there is no page for increasing the limit. Looks like the docs are outdated or sth, but there is just no option to increase anything. The docs is pointing to pages and buttons which do not exist.
Even when I managed to get into some page with Action limits (by changing URL address because in my personal account, I see that spending limits page), I changed it from 0 to 1000 but it still does not work.
Any help appreciated, I am stuck and can't use custom runners at all. Thanks

**Reply by jmodrako** [reply]
Same issue here as @trubi

**Comment by john-lyq** [comment]
` https://github.com/organizations/**<Organization>**/settings/billing/spending_limit   `     Among the actions, increase the spending limit, and then wait for 1 minute for the runner to become ready.
